server/src/main/java/org/opensearch/cluster/decommission/NodeDecommissionedException.java

```diff
@@ -0,0 +1,31 @@
+/*
```

Comment:
No Need to review this file. 
Taken from Decommission PR: https://github.com/opensearch-project/OpenSearch/pull/4084